packages/xdl/src/ip.js

```diff
@@ -92,7 +99,7 @@ function address(name, family) {
       //
       var addresses = interfaces[nic].filter(function(details) {
         details.family = details.family.toLowerCase();
-        if (details.family !== family || isLoopback(details.address)) {
+        if (details.family !== family || isLoopback(details.address) || isVirtualNetworkAdapter(details.mac)) {
```

Comment:
Since this filters out the interface altogether if it's a VMWare virtual network adapter, will this work for people who are running Expo CLI within a VMWare virtual machine?

Maybe we could tweak the [`priority`](https://github.com/expo/expo-cli/blob/371be04175a676ae789544836ada6421a5d95937/packages/xdl/src/ip.js#L9) function instead, so that other NICs would be sorted before them, but if no regular NIC exists, the virtual NIC address could still be returned?

Reply:
Theoretically it should work both on virtualized systems and not.
I tried it practically (on a vm with VMware Workstation and Win10 as OS) and the result was what I expected basing on my knowledge. Launching an `ipconfig /all` on the vm it gives this result:
![cattura](https://user-images.githubusercontent.com/17990410/50992788-60445480-1518-11e9-8d30-08db6c53651f.PNG)

Virtualization is a job made from hypervisors which virtualize hardware, so NICs too.
For this, the virtual NIC you see on the container, will be virtualized once you boot the guest. So, basically, from the container you see a virtual NIC - having the VMware OUI - but from guest you see a virtualized NIC - having Intel OUI in my test.

The cool thing is that it should work even on a virtualized machine running on a virtualized machine, in a Russian dolls style.
Then fix affects only virtualized NICs created with VMware, which are the ones created using VMware hypervisor.